pages/blog/blog-post-template.html

```diff
@@ -34,13 +34,16 @@ <h3>Placeholder Section Title</h3>
                 If you need more sections, add more h3 tags below this one.
             </p>
         </article>
+
+        <footer-component></footer-component>
+        <gdpr-component></gdpr-component>
```

Comment:
This one is fairly straightforward 😄

I've just moved our HTML components to the bottom of our file. With these two, it doesn't matter which one is set up/added first as our 'gdpr-component' has a z-index value so it will always be displayed on top of any content (if the content has a lower z-index value - in this case, we have none set so 'gdpr-component' will always be on top of everything until closed)